css/components/buttons.css

```diff
@@ -1,74 +1,138 @@
 .btn {
-	outline: 0;
-	cursor: pointer;
-	border: 0;
-	padding: 0.5rem 0.8rem;
+  outline: 0;
+  cursor: pointer;
+  border: 0;
+  padding: 0.5rem 0.8rem;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+}
+
+.btn:active {
+  transform: scale(1.09);
 }
 
 .btn-rounded {
-	border-radius: 0.5rem;
+  border-radius: 0.5rem;
 }
 
 .link {
```

Comment:
I'm supposing that this link is specific for buttons.
So isn't it will be good to give the class name as btn-link.

Reply:
it can be used as a link like normal anchor tag

Reply:
Oh okay, so it's like universal class. It can be in utility folder.

Reply:
i'm not following 7-1 strucutre. May be I will change it link-btn

Reply:
Rest everything is fine.